sdk/python/feast/feature_store.py

```diff
@@ -1437,7 +1438,7 @@ def _read_from_online_store(
         provider: Provider,
         requested_features: List[str],
         table: FeatureView,
-    ) -> List[Tuple[List[Timestamp], List["FieldStatus.ValueType"], List[Value]]]:
+    ) -> List[Tuple[List[Timestamp], List[FieldStatusValue], List[Value]]]:
```

Comment:
Trying to resolve https://github.com/feast-dev/feast/issues/2249 but I can revert.